test/xcdr/appendable.cpp

```diff
@@ -185,6 +185,79 @@ void deserialize(
 } // namespace fastcdr
 } // namespace eprosima
 
+struct MutableElement
+{
+public:
+
+    MutableElement() = default;
+
+    MutableElement(
+            uint8_t value)
+    {
+        value1 = value;
+        value2 = value;
+    }
+
+    bool operator ==(
+            const MutableElement& other) const
+    {
+        return value1 == other.value1 && value2 == other.value2;
+    }
+
+    //! First being serialized.
+    uint32_t value1 {0};
+
+    //! Third being serialized.
+    uint16_t value2 {0};
```

Comment:
Probably the comments have directly been copied and not updated to this new test.

Reply:
Done in 8099200